nb/api/seam.md

```diff
@@ -0,0 +1,46 @@
+### seam()
+Parameter|Default|Type
+---|---|---
+...selections||Shapes to select the areas to seam.
+
+Add non-geometric edges where the selections intersect the shape.
```

Comment:
I understand what this does from the notes/examples, but I don't really understand why one would want to do it 😛

Reply:
Yeah, it has relatively limited application.